lib/ignored-paths.js

```diff
@@ -146,6 +163,31 @@ class IgnoredPaths {
                 this.baseDir = path.dirname(path.resolve(options.cwd, ignorePath));
                 this.addIgnoreFile(this.ig.custom, ignorePath);
                 this.addIgnoreFile(this.ig.default, ignorePath);
+            } else {
+                try {
+
+                    // if the ignoreFile does not exist, check package.json for eslintIgnore
+                    const packageJSONPath = findPackageJSONFile(options.cwd);
+
+                    if (packageJSONPath) {
+                        let packageJSONOptions;
+
+                        try {
+                            packageJSONOptions = JSON.parse(fs.readFileSync(packageJSONPath, "utf8"));
+                        } catch (e) {
+                            debug("Could not read package.json file to check eslintIgnore property");
+                        }
+
+                        if (packageJSONOptions.eslintIgnore && Array.isArray(packageJSONOptions.eslintIgnore)) {
```

Comment:
Should we throw an error if `packageJSONOptions.eslintIgnore` is not an array? Seems like it would silently not do anything in this case.